lib/rest/Twilio.ts

```diff
@@ -0,0 +1,518 @@
+ * @property (Twilio.Api.V2010.AccountContext.TokensList) tokens - tokens resource
+ * @property (Twilio.Api.V2010.AccountContext.TranscriptionsList) transcriptions - transcriptions resource
+ * @property (Twilio.Api.V2010.AccountContext.UsageList) usage - usage resource
+ *
+ * @param {string} username -
+ *          The username used for authentication. This is normally account sid, but if using key/secret auth will be the api key sid.
+ * @param {string} password -
+ *          The password used for authentication. This is normally auth token, but if using key/secret auth will be the secret.
+ * @param {opts} [opts] - The options argument
+ *
+ * @returns {Twilio} A new instance of Twilio client
+ */
+/* jshint ignore:end */
+
+
+export class Twilio extends BaseTwilio {
+  username: string
+  password: string
+  opts: opts
+  _numbersv2: any
+  _chatv1: any
+  _conversationsv1: any
+  _bulkexportsv1: any
+  _insightsv1: any
+  _trusthubv1: any
+  _flexv1: any
+  _videov1: any
+  _frontlinev1: any
+  _trunkingv1: any
+  _notifyv1: any
+  _pricingv2: any
+  _api: any
+  _studiov1: any
+  _supersimv1: any
+  _monitorv1: any
+  _verifyv2: any
+  _eventsv1: any
+  _voicev1: any
+  _ipmessagingv1: any
+  _mediav1: any
+  _faxv1: any
+  _serverlessv1: any
+  _syncv1: any
+  _studiov2: any
+  _chatv2: any
+  _accountsv1: any
+  _chatv3: any
+  _messagingv1: any
+  _lookupsv1: any
+  _autopilotv1: any
+  _taskrouterv1: any
+  _wirelessv1: any
+  _proxyv1: any
+  _ipmessagingv2: any
+  _pricingv1: any
+
+  constructor(username: string, password: string, opts: opts) {
+    super(username, password, opts);
+
+    //Domains
+    this._numbersv2 = undefined
+    this._chatv1 = undefined
+    this._conversationsv1 = undefined
+    this._bulkexportsv1 = undefined
+    this._insightsv1 = undefined
+    this._trusthubv1 = undefined
+    this._flexv1 = undefined
+    this._videov1 = undefined
+    this._frontlinev1 = undefined
+    this._trunkingv1 = undefined
+    this._notifyv1 = undefined
+    this._pricingv2 = undefined
+    this._api = undefined
+    this._studiov1 = undefined
+    this._supersimv1 = undefined
+    this._monitorv1 = undefined
+    this._verifyv2 = undefined
+    this._eventsv1 = undefined
+    this._voicev1 = undefined
+    this._ipmessagingv1 = undefined
+    this._mediav1 = undefined
+    this._faxv1 = undefined
+    this._serverlessv1 = undefined
+    this._syncv1 = undefined
+    this._studiov2 = undefined
+    this._chatv2 = undefined
+    this._accountsv1 = undefined
+    this._chatv3 = undefined
+    this._messagingv1 = undefined
+    this._lookupsv1 = undefined
+    this._autopilotv1 = undefined
+    this._taskrouterv1 = undefined
+    this._wirelessv1 = undefined
+    this._proxyv1 = undefined
+    this._ipmessagingv2 = undefined
+    this._pricingv1 = undefined
+
+    if (!opts.lazyLoading) {
+        this._numbersv2
+        this._chatv1
+        this._conversationsv1
+        this._bulkexportsv1
```

Comment:
Should see about sorting these. Could just need a `sorted()` wrapper on the list return from `listdir`. See the recent Terraform provider changes in librarian for reference.